api/config/packages/security.yaml

```diff
@@ -23,13 +23,13 @@ security:
                 success_handler: lexik_jwt_authentication.handler.authentication_success
                 failure_handler: lexik_jwt_authentication.handler.authentication_failure
 
-        api:
-            pattern:  ^/api
-            stateless: true
-            provider: entity_provider
-            guard:
-                authenticators:
-                    - lexik_jwt_authentication.jwt_token_authenticator
+        # api:
+        #     pattern:  ^/api
+        #     stateless: true
+        #     provider: entity_provider
+        #     guard:
+        #         authenticators:
+        #             - lexik_jwt_authentication.jwt_token_authenticator
```

Comment:
Why did you delete the auth system ?

Reply:
cause i wasn't able to work with it, as i don't know that much api platform i wasn't able to disable it on routes that i wanted too and i think it'll be the same for the others